README.md

```diff
@@ -123,11 +128,20 @@ Forgeries is available through [CocoaPods](http://cocoapods.org). To install it,
 target 'MyApp_Tests' do
   inherit! :search_paths
 
-  pod 'forgeries'
+  pod 'Forgeries'
   ...
 end
 ```
 
+That will import the core functionality, which is currently just `UIGestureRecognizer` subclasses. For everything else, use:
```

Comment:
Yeah, agreed it's not idea. To be honest I just put them there because I was tired of fighting Xcode and that momentarily quieted things. I'll put them back.